plugins/storage/pyparts/ceph_daemon_checks.py

```diff
@@ -236,11 +236,10 @@ def get_crushmap_mixed_buckets(self):
                 bad_buckets.append(buckets[bid]["name"])
 
         if bad_buckets:
-            msg = ("mixed crush buckets identified (see --storage for more "
-                   "info)")
+            msg = ("mixed crush buckets identified in buckets {}."
+                   .format(bad_buckets))
```

Comment:
We could perhaps go a bit further with this message and allude to what should be checked/fixed e.g.

msg = ("mixed crush bucket types identified in buckets '{}'. This can cause data distribution to become skewed - please check crush map".format(bad_buckets))